content/sensu-enterprise/2.6/integrations/email.md

```diff
@@ -286,3 +289,4 @@ example      | {{< highlight shell >}}"body": "/etc/sensu/email/body_template.er
 [3]:  #templates-attributes
 [4]:  /sensu-core/1.0/reference/events#event-data
 [5]:  https://docs.puppet.com/puppet/latest/lang_template_erb.html
+[6]:  /sensu-core/1.4/reference/clients/#custom-attributes
```

Comment:
I think we need Enterprise docs to reference material for the version of Sensu Core that correlates to the Enterprise version. Here this should be Core 1.0